material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex

```diff
@@ -1,116 +1,240 @@
+       &\posvec{N_o}{Q}& $x\uvecx{n} \plus y\uvecy{n} \plus o\uvecz{n}$
+       \\[1.0pc]
+       &\posvec{N_o}{B_o}& $50\uvecx{n} \plus 0\uvecy{n} \plus 20\uvecz{n}$
+       \\[1.0pc]
+       &\posvec{B_o}{Q} & $\lambda\uvecx{b}$
+       \\[1.0pc]
+       Loop: & \posvec{N_o}{Q}&\posvec{N_o}{B_o}\plus\posvec{B_o}{Q}
+       \\[1.0pc]
+       &$x\uvecx{n} \plus y\uvecy{n}$& $50\uvecx{n} \plus
+       20\uvecz{n} \plus\lambda\uvecx{b}$
+       \\[1.0pc]
+       \end{tabular}
+       \begin{tabular}{c@{}l}
+       \begin{tabular}{lr@{ \equals[\;] }l}
+       dot w/ \uvecx{n}: & x & $ 50 \plus \lambda \uvecx{b} \DotProduct \uvecx{n} $
+       \\[1.0pc]
+       dot w/ \uvecy{n}: & y & $ 0 \plus \lambda \uvecx{b} \DotProduct \uvecy{n} $
+       \\[1.0pc]
+       dot w/ \uvecz{n}: & o & $ 20 \plus \lambda \uvecx{b} \DotProduct \uvecz{n} $
+       \\[1.0pc]
+       \end{tabular}
+       &
+       $\begin{array}{l}
+       \MyLBrace{8.9ex}{scalar system w/ 3 equations, 3 unknowns x,y,$\lambda$}
+       \end{array}$
+       \end{tabular}
+       \end{minipage}
+       \\[0.45pc]
+       }
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\clearpage
-\item
-%\hspace*{0pt}%\textbf{( 12 pts )}
-%\\[-2.7pc]
-\begin{minipage}[t]{0.53\columnwidth}
-The camera body is mounted on a pan-tilt mechanism; the ``pan" part lets the camera rotate ``left-and-right" and the ``tilt" part lets the camera rotate ``up-and-down".
+  \clearpage
+  \item
+  %\hspace*{0pt}%\textbf{( 12 pts )}
+  %\\[-2.7pc]
+  \begin{minipage}[t]{0.53\columnwidth}
+  The camera body is mounted on a pan-tilt mechanism; the ``pan" part
+  lets the camera rotate ``left-and-right" and the ``tilt" part lets
+  the camera rotate ``up-and-down".
+  \\[1.0pc]
+  Let the ``pan" portion of the mechanism be a rigid frame \basis{A}
+  with orthogonal unit vectors \uvecxyz{a}. Frame \basis{A} is initially
+  aligned with \basis{N} but is then rotated with $\plus\uvecz{n}$ sense
+  by an angle $\theta$.
+  (Hint: $\theta$ is the angle between \uvecy{n} and \uvecy{a}.)
+  \\[1.0pc]
+  The ``tilt" portion of the mechanism can be assumed to be the same
+  as the camera body frame \basis{B}, which is initially aligned with
+  \basis{A} but is then rotated with $\plus\uvecy{a}$ sense by an angle
+  $\phi$. (Hint: $\phi$ is the angle between
+  \uvecx{a} and \uvecx{b}.)
+  \\[1.0pc]
+  Please \textbf{\underline{write two rotation tables}}: one relating
+  \basis{N} and \basis{A} and one relating \basis{A} and \basis{B}.
+  \\[0.25pc]
+  Then \textbf{\underline{explain}} how to obtain the rotation
+  table relating frame \basis{N} and \basis{B}
+  (don't actually compute it).
+  \end{minipage}
+  \hfill
+  \begin{minipage}[t]{0.37\columnwidth}
+  \flushright
+  \vspace*{0pt}
+  \includegraphics[width=7cm]{camera_pan_tilt.jpg}
+  \end{minipage}
+\Solution{}{0.99\linewidth}{
+  \begin{minipage}{0.34\linewidth}
+    \minipageTopAnchor
+    \center
+    \rotationTable{N}{A}
+    {\cos\theta}{\minus\sin\theta}{0}
```

Comment:
hmm... it looks weird with so much space after the minus sign. Maybe just use `-` for these.